src/ar/anime4up/src/eu/kanade/tachiyomi/animeextension/ar/anime4up/Anime4Up.kt

```diff
@@ -211,6 +241,15 @@ class Anime4Up : ConfigurableAnimeSource, ParsedAnimeHttpSource() {
     }
 
     // ============================= Utilities ==============================
+    private inline fun <A, B> Iterable<A>.parallelCatchingFlatMap(crossinline f: suspend (A) -> Iterable<B>): List<B> =
```

Comment:
Isn't this already in the extension-lib 14? Use that instead of duplicating the code

Reply:
import back `parallelCatchingFlatMapBlocking` and use that one. you have deleted it above